### PR TITLE
Added basic contextual menu

### PR DESCRIPTION
**Pull request by barrymcgee**
Storybook: http://localhost:61502/?path=/docs/contextualmenu--default-story

### CODE REVIEW

File: src/components/ContextualMenu/ContextualMenu.test.js [hunk 1]
```diff
@@ -0,0 +1,18 @@
+import { shallow } from "enzyme";
+import React from "react";
+
+import ContextualMenu from "./ContextualMenu";
+
+describe("Contextual Menu ", () => {
+  it("renders instance but not visible", () => {
+    const wrapper = shallow(<ContextualMenu>Test content</ContextualMenu>);
+    expect(wrapper).toMatchSnapshot();
```
Comment (hatched) [inline src/components/ContextualMenu/ContextualMenu.test.js:9]: Snapshots in this case aren't the best idea, you should have an assertion for what you're actually testing.

Comment (squidsoup) [reply src/components/ContextualMenu/ContextualMenu.test.js:9]: Agree with @hatched, snapshot tests in isolation are not sufficient, but okay to have _both_ I think.

File: src/components/ContextualMenu/ContextualMenu.test.js [hunk 2]
```diff
@@ -0,0 +1,22 @@
+import { shallow } from "enzyme";
+import React from "react";
+
+import ContextualMenu from "./ContextualMenu";
+
+describe("Contextual Menu ", () => {
+  it("renders instance but not visible", () => {
```
Comment (anthonydillon) [inline src/components/ContextualMenu/ContextualMenu.test.js:7]: Are you missing a snapshot test?

File: src/components/SearchAndFilter/SearchAndFilter.js [hunk 3]
```diff
@@ -0,0 +1,34 @@
...
+
+import SearchBox from "../SearchBox";
+
+const SearchAndFilter = ({ externallyControlled = false, onChange }) => {
+  const [searchTerm, setSearchTerm] = useState("");
+
+  const searchOnChange = (searchTerm) => {
+    setSearchTerm(searchTerm);
+    if (externallyControlled) {
+      onChange(searchTerm);
```
Comment (anthonydillon) [inline src/components/SearchAndFilter/SearchAndFilter.js:12]: You should check onChange exists here otherwise you will get unexpected errors.

**Pull request merged**